dashlane-project/modules/nfccardreader/src/main/java/com/github/devnied/emvnfccard/model/enums/CountryCodeEnum.java

```diff
@@ -333,7 +333,7 @@ public enum CountryCodeEnum implements IKeyEnum {
 
 	
 
-	GG("Guemsey", "GGY", 831),
+	GG("Guernsey", "GGY", 831),
```

Comment:
This is fairly notable.

I'm pretty sure it's a reference to https://en.wikipedia.org/wiki/Guernsey